js-api-doc/value/color.d.ts

```diff
@@ -12,117 +90,456 @@ export class SassColor extends Value {
+   * recommended [CSS Gamut Mapping Algorithm][css-mapping] to map out-of-gamut
+   * colors into the desired gamut with as little perceptual change as possible.
+   *
+   * [css-mapping]: https://www.w3.org/TR/css-color-4/#css-gamut-mapping-algorithm
+   */
+  toGamut(space?: KnownColorSpace): SassColor;
+
+  /**
+   * A list of this color's channel values (excluding alpha), with [missing
+   * channels] converted to `null`.
+   *
+   * [missing channels]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   */
+  get channelsOrNull(): List<number | null>;
+
+  /**
+   * A list of this color's channel values (excluding alpha), with [missing
+   * channels] converted to `0`.
+   *
+   * [missing channels]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   */
+  get channels(): List<number>;
+
+  /**
+   * Returns the value of a single specified `channel` of this color, with
+   * [missing channels] converted to `0`.
+   *
+   * [missing channels]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   *
+   * @throws `Error` if `channel` is not `alpha` or a channel in this color's
+   * space.
+   */
+  channel(channel: ChannelName): number;
+
+  /**
+   * Returns the value of a single specified `channel` of this color after
+   * converting this color to the specified `space`, with [missing channels]
+   * converted to `0`.
+   *
+   * [missing channels]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   *
+   * @throws `Error` if `channel` is not `alpha` or a channel in `space`.
+   */
+  channel(
+    channel: ChannelNameHSL | 'alpha',
+    options: {space: ColorSpaceHSL}
+  ): number;
+  channel(
+    channel: ChannelNameHWB | 'alpha',
+    options: {space: ColorSpaceHWB}
+  ): number;
+  channel(
+    channel: ChannelNameLab | 'alpha',
+    options: {space: ColorSpaceLab}
+  ): number;
+  channel(
+    channel: ChannelNameLCH | 'alpha',
+    options: {space: ColorSpaceLCH}
+  ): number;
+  channel(
+    channel: ChannelNameRGB | 'alpha',
+    options: {space: ColorSpaceRGB}
+  ): number;
+  channel(
+    channel: ChannelNameXYZ | 'alpha',
+    options: {space: ColorSpaceXYZ}
+  ): number;
 
   /** This color's alpha channel, between `0` and `1`. */
   get alpha(): number;
 
   /**
-   * Changes one or more of this color's RGB channels and returns the result.
+   * Returns a boolean indicating whether a given channel value is a [missing
+   * channel].
+   *
+   * [missing channel]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   */
+  isChannelMissing(channel: ChannelName | 'alpha'): boolean;
+
+  /**
+   * A boolean indicating whether this color's alpha channel is a [missing
+   * channel].
+   *
+   * [missing channel]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   */
+  get isAlphaMissing(): boolean;
```

Comment:
On consideration, I'm not sure this is necessary if the user can just write `isChannelMissing('alpha')` anyway.